scriptmodules/emulators/residualvm.sh

```diff
@@ -29,7 +29,9 @@ function sources_residualvm() {
 }
 
 function build_residualvm() {
-    ./configure --enable-vkeybd --enable-release --disable-debug --enable-keymapper --prefix="$md_inst"
+    export SDL_CONFIG=sdl2-config
+    export LDFLAGS="-L/opt/vc/lib"
```

Comment:
these shouldn't be exported to the environment - you can set them just before `./configure`